projects/packages/my-jetpack/_inc/components/product-cards-section/anti-spam-card.jsx

```diff
@@ -22,7 +22,7 @@ export const AntiSpamIcon = () => (
 );
 
 const AntiSpamCard = ( { admin } ) => {
-	const { status, activate, deactivate, detail, isFetching } = useProduct( 'anti-spam' );
+	const { slug, status, activate, deactivate, detail, isFetching } = useProduct( 'anti-spam' );
 	const { name, description } = detail;
```

Comment:
```suggestion
	const { status, activate, deactivate, detail, isFetching } = useProduct( 'anti-spam' );
	const { name, description, slug } = detail;
```

Reply:
oh thanks

Reply:
> The idea behind using the detail object is to collect all product data relevant to it while exposing helpers and not-product data in the root. Thus

Is `slug` a "detail" though? It's more the key we use to access the detailed information. So we actually know it in advance when calling `useProduct()`. Anyways, working on a commit to consume it from `detail`.

Reply:
Addressed in [a40e2b2](https://github.com/Automattic/jetpack/pull/22756/commits/a40e2b2e0af25e1aad67f10d94d578e4cdd0bfe3)